src/components/Link.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import classNames from "classnames";
+import PropTypes from "prop-types";
+
+import ComponentRegister, { LinkComponent } from "../ComponentRegister";
+
+import "lbh-frontend/lbh/core/_links.scss";
+
+/**
+ * Generates a Link component that's a wrapper for the Link attribute of
+ * ComponentRegister
+ * @param {string} [className] - A string of class names
```

Comment:
```suggestion
 *
 * @param {string} [className] - A string of class names
```

Whitespace is good and helps readability by creating sections in your text (and code).

Reply:
I meant this in general :).